proxy/src/main/java/net/md_5/bungee/BungeeCord.java

```diff
@@ -296,7 +298,9 @@ public void operationComplete(ChannelFuture future) throws Exception
                     .option( ChannelOption.SO_REUSEADDR, true ) // TODO: Move this elsewhere!
                     .childAttr( PipelineUtils.LISTENER, info )
                     .childHandler( PipelineUtils.SERVER_CHILD )
-                    .group( eventLoops )
+                    .childOption(ChannelOption.WRITE_BUFFER_HIGH_WATER_MARK, 1024 * 1024 * 10)
```

Comment:
This is from a patch from @Iceee that reduces GC churn for certain classes of attacks against BungeeCord.